packages/components/src/InlineLabel/InlineLabel.module.css

```diff
@@ -12,7 +12,7 @@
 }
 
 .large {
-  padding: calc(var(--space-small) * 1.25) calc(var(--space-small) * 1.5);
+  padding: calc(var(--space-small) * 1.25) var(--space-slim);
```

Comment:
It is at least slightly funny that we _still_ have a calc left in here, but the 10px value is much less widespread